gazetteer-spec/0.0/README.md

```diff
@@ -1,8 +1,8 @@
-# Gazetteer specification
+# Gazetteer 0.0
```

Comment:
Should this match 1.0 or nah?

Reply:
Confirmed in chat that this is OK. Just documenting the 0.0 spec for prosperity.